src/lib/common/errorMessages.h

```diff
@@ -62,7 +62,7 @@
 #define ERROR_DESC_BAD_REQUEST_INVALID_JTYPE_ENTTYPE  "Invalid JSON type for entity type"
 #define ERROR_DESC_BAD_REQUEST_INVALID_JTYPE_SCOPE    "invalid JSON type for scope value: must be string"
 #define ERROR_DESC_BAD_REQUEST_INVALID_ATTRSFORMAT    "invalid attrsFormat, accepted values: legacy, normalized, keyValues, values"
-#define ERROR_DESC_BAD_REQUEST_INVALID_STATUS         "status is not valid: it has to be either active or inactive"
+#define ERROR_DESC_BAD_REQUEST_INVALID_STATUS         "status is not valid: it has to be either active or inactive or oneshot"
```

Comment:
active or inactive or oneshot -> active, inactive or oneshot

("... or ... or ... or" works fine for machines in programming languages but I think "... , ..., ..., or ...." is better for humans ;)